.remoteRotator.toml

```diff
@@ -12,19 +12,19 @@ port = 3333
 [http]
 enabled = true
 host = "127.0.0.1"
-port = 7070
+port = 7071
```

Comment:
I would prefer to not modify the default values in the `.remoteRotator.toml` config file

Reply:
Hi Tobi,

Line 384 removed! 
And Ok about the configuration file. 

Rgds, Pablo EA4TX